hi-ml-histopathology/src/histopathology/configs/classification/BaseMIL.py

```diff
@@ -92,10 +92,12 @@ class BaseMIL(LightningContainer):
 
     # Outputs Handler parameters:
     save_output_slides: bool = param.Boolean(True, doc="a boolean parameter to enable saving heatmaps and thumbnails.")
-    k_slides: int = param.Integer(10, bounds=(0, None),
-                                  doc="Number of slide to save top and bottom tiles. Defaults 10.")
-    k_tiles: int = param.Integer(10, bounds=(0, None),
-                                 doc="Number of tiles to save top and bottom tiles. Defaults 10.")
+    n_top_slides: int = param.Integer(10, bounds=(0, None), doc="Number of slides to select when saving top and bottom "
```

Comment:
sorry for nitpicking on naming: "n_tiles" is a slight improvement over "k_tiles", but still leaves some amount of ambiguity. I'd go for "num_" as the prefix rather than "n"